style.css

```diff
@@ -0,0 +1,81 @@
+
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@500;600;700&display=swap");
+
+body {
+  background-color: #e5e5e5;
+  position: absolute;
+  width: 375px;
+  height: 718px;
+  margin: 15%;
+}
+
+.home-page {
+  background-color: #fff;
+  border-radius: 0 0 0 5%;
+  display: flex;
+  flex-direction: column;
+  height: 600px;
+}
+
+.log {
+  height: 30px;
+  width: 30px;
+  display: flex;
+  align-items: center;
+}
+
+.hamburger {
+  display: flex;
+  justify-content: space-between;
+  margin: 24px 12px 0 10px;
+  padding: 0 48px 0 48px;
+}
+
+#home {
+  background-image: url("/img/imagee.png");
```

Comment:
- [ ] Well done 👏🏻 you made the changes as per the previous code reviewer's request, but I can see that you have a typo which is creating the problem to make your background image visible to the screen. Kindly have a look at the below suggestion to fix the issue.

``` suggestion
  background-image: url("./img/imagee.png");
```